### PR TITLE
LPD-39086 - Sort by Last Login Date appears to be sorting by Last Name

### PR DESCRIPTION
**Pull request by brbalazs**
https://liferay.atlassian.net/browse/LPD-39086

### CODE REVIEW

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by brbalazs** [comment]
ci:test:sf

File: portal-kernel/src/com/liferay/portal/kernel/util/comparator/UserLastLoginDateComparator.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+ */
+
+package com.liferay.portal.kernel.util.comparator;
+
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.util.DateUtil;
+import com.liferay.portal.kernel.util.OrderByComparator;
+
+/**
+ * @author Brian Wing Shun Chan
```
Comment (smottal) [inline portal-kernel/src/com/liferay/portal/kernel/util/comparator/UserLastLoginDateComparator.java:13]: mmm

**Review by smottal — CHANGES_REQUESTED** [review]

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by brbalazs** [comment]
ci:test:relevant

**Comment by brbalazs** [comment]
ci:stop

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by brbalazs** [comment]
ci:report:60493199

**Comment by smottal** [comment]
ci:forward:force

**Pull request closed without merging**